docker/worker/Pipfile

```diff
@@ -8,7 +8,7 @@ google-cloud-pubsub = "*"
 google-cloud-datastore = "*"
 google-cloud-logging = "*"
 google-cloud-ndb = "*"
-pygit2 = "*"
+pygit2 = "==1.5.0"
```

Comment:
Can you add a comment here explaining why we need this? And possibly also a TODO.